app/services/log.js

```diff
@@ -50,6 +50,18 @@ if (ENABLE_SLACK_ALERTING) {
 }
 
 const logger = {
+  trackExecutionTime: function (jobName, timeTaken, status) {
+    if (appInsightsClient) {
+      appInsightsClient.trackDependency({
+        dependencyTypeName: 'myDependency',
```

Comment:
could this be named to something like `workerTaskDependency`? In the future we may put dependency tracking around the database calls and want to name it something else

Reply:
That tracking is already there by default - named postgres.
I definitely need to rename this though!